src/lib/Microsoft.Health.Common/Service/ParallelTaskWorker.cs

```diff
@@ -66,7 +66,7 @@ await consumer.Completion
                     {
                         if (!exceptions.IsEmpty)
                         {
-                            throw new AggregateException(exceptions);
+                            throw new SimpleAggregateException(exceptions);

```

Comment:
Another alternative is we log all the individual exceptions here.  That still might be a good thing to do even with this change to ensure we have discrete messages for every error for readability.

Reply:
This will also ensure each exception's stack trace is recorded which we are currently missing.

Reply:
I had considered doing that but wasn't sure if we wanted to report 1 exception per event or 1 per batch like we do now. I'm fine with reporting each exception in order to gain the stack trace and removing this new exception class as it won't be used

Reply:
Done